spec/kong_tests.conf

```diff
@@ -4,10 +4,10 @@ proxy_listen = 0.0.0.0:9000
 proxy_listen_ssl = 0.0.0.0:9443
 cluster_listen = 0.0.0.0:9946
 cluster_listen_rpc = 127.0.0.1:9373
-ssl_cert = ../spec/fixtures/kong_spec.crt
-ssl_cert_key = ../spec/fixtures/kong_spec.key
-dnsmasq = off
-dns_resolver = 8.8.8.8
+ssl_cert = spec/fixtures/kong_spec.crt
+ssl_cert_key = spec/fixtures/kong_spec.key
+dnsmasq = on
+dnsmasq_port = 9053
```

Comment:
Tests should not use dnsmasq, to keep the running Kong as light as possible (1 worker + Serf because mandatory). The bare minimum. Only enable dnsmasq for tests requiring it, following the provided helper methods to prepare the Nginx prefix (configuration) with settings given as a Lua table in the `setup` phase of these tests.